### PR TITLE
101-fix-gender-dropdown-style-and-make-it-selectable

### PR DESCRIPTION
**Pull request by simonac2-gif**
This is the right pull request. For 101 not 59

### CODE REVIEW

File: components/Start/elements/Details.js [hunk 1]
```diff
@@ -2,14 +2,20 @@ import _Button from "./_Button";
 import _Input from "./_Input";
 import _Header from "./_Header";
 import _Divider from "./_Divider";
+import RadioForm from "react-native-simple-radio-button";
```
Comment (softwarebyze) [inline components/Start/elements/Details.js:5]: This library isn't in our package.json and theres no changes to our package json in this PR so I'm getting an error that it's not found. We should be able to implement this ourselves without pulling in a library

File: components/Start/elements/Details.js [hunk 2]
```diff
@@ -2,14 +2,20 @@ import _Button from "./_Button";
...
 import Styles from "../styles/Styles";
 import TERMS from "../../../settings/Terms";
 import { useNavigation } from "@react-navigation/native";
 import { SafeAreaView } from "react-native-safe-area-context";
 import { Ionicons } from "@expo/vector-icons";
 import PhoneInput from "react-native-phone-input";
-import { Text, View, TouchableOpacity } from "react-native";
+import {
+  Text,
+  TouchableNativeFeedback,
```
Comment (softwarebyze) [inline components/Start/elements/Details.js:15]: not sure why [TouchableNativeFeedback](https://reactnative.dev/docs/touchablenativefeedback) is being imported

**Review by softwarebyze — CHANGES_REQUESTED** [review]

**Review by softwarebyze — APPROVED** [review]

**Comment by softwarebyze** [comment]
https://github.com/softwarebyze/Places/assets/77241170/d9714fc4-ce84-4784-b9d1-cbb2cfbd03f8

**Comment by softwarebyze** [comment]
https://github.com/softwarebyze/Places/assets/77241170/3d8f64fd-7185-4df1-a72e-a8c4313fd8ba

**Review by softwarebyze — APPROVED** [review]

**Comment by softwarebyze** [comment]
https://github.com/softwarebyze/Places/assets/77241170/1eddd642-8252-4c30-87bc-fb1f1a1d6e8b

**Comment by simonac2-gif** [comment]
IOS and Android differences with code

**Pull request merged**